module-2/lab-mongodb/Untitled.ipynb

```diff
@@ -5356,18 +5356,18 @@
   },
```

Comment:
- All the companies that have more than 5000 employees. Limit the search to 20 companies and sort them by number of employees. Try to get the name and the number of employees as the result of the query. The id field is useless in these queries. You can use the projection statement in order to filter the id field. 
- "All the companies that have a null type of value on the category_code field". Null is not the same as None. I would query both separately. I encourage you to ask Marc the difference between Null and undefined in JavaScript.
- Remember to ask Marc to solve question number 12!!
- I invite you to explore the aggregate () function in mongo, similar to groupby. Play with this data set to get some aggregations from Mongodb.
- Try to complete this exercise when you have some free time.